package.json

```diff
@@ -24,12 +24,13 @@
   },
   "homepage": "https://github.com/3box/3box-api#readme",
   "dependencies": {
-    "3box-orbitdb-plugins": "^1.0.6",
+    "3box-orbitdb-plugins": "3box/3box-orbitdb-plugins#feat/orbit-23",
     "3id-resolver": "^0.0.4",
     "analytics-node": "^3.3.0",
     "aws-sdk": "2.414.0",
     "axios": "^0.19.0",
-    "cids": "0.7.1",
+    "bunyan": "^1.8.12",
```

Comment:
What's this thing that was added?

Reply:
logging, missing dependency from earlier release, or now missing